src/index.css

```diff
@@ -59,15 +68,42 @@ nav a {
 
 .inner-container {
   display: grid;
-  grid-template-rows: 1fr 1fr 1fr;
+  grid-template-columns: repeat(3, minmax(0, 1fr));
+  gap: 15px;
   border: 1px solid #dbdada;
-  padding: 5px 10px;
+  padding: 20px;
+  justify-content: center;
+}
+
+.box {
+  padding: 5px 15px;
+  align-items: center;
+}
+
+.grey {
+  color: grey;
+}
+
+.each-book {
+  display: flex;
```

Comment:
- [ ] Here I can see you have used Flexbox and you have repeated it many times in your file. Kindly keep your code  [DRY](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) by using a reusable CSS class and then applying that classname to the elements that use this property.